binaries/scoot-snapshot-db/main.go

```diff
@@ -36,11 +37,13 @@ func main() {
 
 	inj := &injector{}
 	cmd := cli.MakeDBCLI(inj)
-	if err := cmd.Execute(); err != nil {
-		removeTemp()
+	err = cmd.Execute()
+	if dbTempDir != "" {
+		os.RemoveAll(dbTempDir)
```

Comment:
when will dbTempDir not be ""?

Reply:
If tempdir creation fails. Otherwise it will do is.RemoveAll("") - which I'm not sure exactly what that would do. Nothing? Or delete everything in your cwd?

Reply:
checked the os pkg implementation and "" is special cased to do nothing and return nil, so removing the check.